docs/root/intro/arch_overview/ssl.rst

```diff
@@ -30,6 +30,23 @@ Underlying implementation
 Currently Envoy is written to use `BoringSSL <https://boringssl.googlesource.com/boringssl>`_ as the
 TLS provider.
 
+.. _arch_overview_ssl_fips:
+
+FIPS
+----
+
+BoringSSL can be built in a
+`FIPS-compliant mode <https://boringssl.googlesource.com/boringssl/+/master/crypto/fipsmodule/FIPS.md>`_,
```

Comment:
That's because it's not Markdown :) I double-checked that the generated document looks fine, and that all links resolve correctly, but thanks for paying extra attention to it.